src/Network/Mattermost/Types.hs

```diff
@@ -1133,6 +1133,7 @@ instance A.ToJSON Reaction where
 data PreferenceCategory
   = PreferenceCategoryDirectChannelShow
   | PreferenceCategoryGroupChannelShow
+  | PreferenceCategoryFavoriteChannelShow
```

Comment:
Since the preference is for whether a channel is a favorite (not whether it is shown), this should be `PreferenceCategoryFavoriteChannel`.

Reply:
fixed in the commit `426e0f6`